Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM node:5.5-slim
-MAINTAINER Michael R. Bernstein <mrb@codeclimate.com>
```

Comment:
I think we still want a MAINTAINER instruction but it should be in the new label form:

```dockerfile
LABEL maintainer="Code Climate <hello@codeclimate.com>"
```

Reply:
Went ahead and added that 👍